### PR TITLE
Fixes to include all options - MAJOR, MINOR, PATCH, PRE-RELEASE

### PR DESCRIPTION
**Pull request by cmario92**
In my case, I wanted to cover the scenarios for all semver options - MAJOR, MINOR, PATCH, PRERELEASE.

I am using the below action config in my project:
image

The reason why I am defining `rc-wording: ''` is because even tho I am setting the `default: prerelease` and none of the `minor-wording, major-wording, patch-wording` matches, there is a default message for rc-wording
image

If the above are not included in the commit message, version bump is skipped.

I also added some comments for future contributors.

### CODE REVIEW

**Comment by cmario92** [comment]
Flow is to check first for:
* Major wordings
* Minor wordings
* Patch wordings
* Prerelease wordings

**Comment by cmario92** [comment]
WA for #104

File: index.js [hunk 1]
```diff
@@ -16,37 +16,52 @@ Toolkit.run(async (tools) => {
...
-    tools.exit.success('No action necessary!');
+    tools.exit.success('No action necessary because we found a previous bump!');
     return;
   }
 
+  // input wordings for MAJOR, MINOR, PATCH, PRE-RELEASE
   const majorWords = process.env['INPUT_MAJOR-WORDING'].split(',');
   const minorWords = process.env['INPUT_MINOR-WORDING'].split(',');
+  // patch is by default empty, and '' would always be true in the includes(''), thats why we handle it separately
+  const patchWords = process.env['INPUT_PATCH-WORDING'] ? process.env['INPUT_PATCH-WORDING'].split(',') : null;
```
Comment (phips28) [inline index.js:36]: @cmario92 this was a missing piece to not break existing default settings with patch.

Comment (phips28) [reply index.js:36]: can you test this with your config please? then we can merge

Comment (cmario92) [reply index.js:36]: I tested both with patch wording and generic wording commit messages to trigger the prc in my case.

all working as expected!

**Pull request merged**